### ISSUE TITLE
ローカライズ項目の追加方法をドキュメントにする

### ISSUE THREAD

**Issue by AyaMorisawa** [misskey-dev/misskey#3240]

**Comment by 4ioskd** [comment]
@robflop 
is this document needed for crowdin??

**Comment by robflop** [comment]
This is about how to add new strings to localize, correct? So how to make more strings for translators show up on Crowdin.

**Comment by 4ioskd** [comment]
I think this is about how to add new languages,strings and so on in order to localize misskey.
But in Crowdin, there is a help page "Knowledge Base" ( https://support.crowdin.com/ ).
So, we needn't write a document how to do suck like that.

**Comment by robflop** [comment]
Yeah, how to use Crowdin can just be read about in their own documentation. It's not something we'd need to document for Misskey, I'd say.
